src/json-api-client.coffee

```diff
@@ -101,9 +102,9 @@ class JSONAPIClient extends Model
     if attributeTypeName?
       type._links[attributeName].type = attributeTypeName
 
-  handleError: ->
+  handleError: (err) ->
     # Override this as necessary.
-    Promise.reject arguments...
+    Promise.reject err
```

Comment:
Previously this rejected with the arguments passed to the handler, so I'm not sure what this change does?